.travis.yml

```diff
@@ -1,15 +1,14 @@
-sudo: true
-dist: trusty
+os: linux
 
-language: python
+language: python 
 
-os:
-    - linux
+python:
+    - "2.7"
+    - "3.8"
 
 env:
-    matrix:
-        - TASK=lint
```

Comment:
Wasn't aware that this goes here without the matrix.

Reply:
(`matrix` is EOL in favour of `jobs`)